domenetjenester/vedtak/src/main/java/no/nav/foreldrepenger/domene/vedtak/migrering/MigrerManglendeForespørslerTjeneste.java

```diff
@@ -79,15 +83,19 @@ public class MigrerManglendeForespørslerTjeneste {
                 }
             }
 
-            var arbeidsgivereMedRefusjon = inntektsmeldingTjeneste.hentInntektsmeldinger(BehandlingReferanse.fra(behandling), stp.getUtledetSkjæringstidspunkt()).stream()
-                    .filter(inntektsmelding -> !inntektsmelding.kommerFraArbeidsgiverPortal() && inntektsmelding.getArbeidsgiver().getErVirksomhet())
-                    .filter(inntektsmelding -> inntektsmelding.getRefusjonBeløpPerMnd() != null && inntektsmelding.getRefusjonBeløpPerMnd().getVerdi().compareTo(BigDecimal.ZERO) > 0)
-                    .map(inntektsmelding -> new OrganisasjonsnummerDto(inntektsmelding.getArbeidsgiver().getOrgnr()))
-                    .toList();
+            var arbeidsgivereMedRefusjon = inntektsmeldingTjeneste.hentInntektsmeldinger(BehandlingReferanse.fra(behandling),
+                    stp.getUtledetSkjæringstidspunkt())
+                .stream()
+                .filter(inntektsmelding -> !inntektsmelding.kommerFraArbeidsgiverPortal() && inntektsmelding.getArbeidsgiver().getErVirksomhet())
+                .filter(inntektsmelding -> inntektsmelding.getRefusjonBeløpPerMnd() != null
```

Comment:
Dette vil vel bety at de som i dag ikke krever refusjon ikke vil få forepørsel? Så hvis de i fremtiden vil be om refusjon via vår portal vil de måtte sende AGI?

Reply:
Vi har jo sett at slike tilfeller har ført til at de har sendt AGI allerede